### DISCUSSION TITLE
tailwind width / height 설정 시 기본 크기

### DISCUSSION THREAD

**Discussion by iamjinpark** [FRONTENDSCHOOL8/popcorn-euid#41] [Q&A] [answered]
브라우저 기본 크기인 16px 기준으로 나눠서 설정해야 하는지.. 아니면 따로 기준이 있는것인지 모르겠네요..
일단 tailwind.config 파일에는 폰트사이즈랑 보더크기밖에 없고 기존 크기에 16px을 나눠서 적용해도 맞지 않아서 질문드립니다.

제가 설정하고 싶은건 width - 85 height - 30 top - 261 left - 118 입니다.
테일윈드 적용시 w-? h-? mt-? ml-? 인가요.. 여러 방면으로 시도해도 안되서 질문 남깁니다..!

**Comment by asdfqaz74** [comment, marked as answer]
1)
현재 따로 적용중인 브라우저 폰트 크기는 없고
기본 크기인 16pt 을 따르고 있습니다.

tailwind.config.js 를 보시면
base: '0.875rem' 
라는 것을 보실 수 있으실텐데 14/16 을 하면 나오는 값입니다.

따라서 14px 을 base 로한 custom fontSize 를 쓰고 있습니다.
현재 쓰고있는 fontSize 를 보면

fontSize: {
      xs: '0.625rem', 10pt
      sm: '0.6564375162124634rem', 10.503pt
      base: '0.875rem', 14pt
      lg: '1.1663750410079956rem', 18.662pt
    },

이렇게 볼 수 있습니다.
따라서 현재 적용되고 있는 기본 브라우저 폰트 크기는 16pt 입니다.

2)
정확하게 지인님이 쓰려는 width - 85 height - 30 top - 261 left - 118 를 테일윈드로 적용시키려면
w-[85/16rem] h-[30/16rem] top-[261/16rem] left-[118/16rem] 이런식으로 쓰시면 됩니다. 

[ ] (대괄호) 는 꼭 있어야 합니다.
[ ] 안에는 계산된 값이 있어야 합니다. ex) w-[85/16rem] >> w-[5.3125rem]

저같은 경우 tailwind에서 자동완성되는 크기에서 최대한 근접한 크기를 쓰고 정확한 크기를 쓰고싶을 때 위와 같은 방법을 쓰고 있습니다.

3)
Tailwind CSS cheatsheet 첨부해드립니다
https://nerdcave.com/tailwind-cheat-sheet

**Reply by iamjinpark** [reply]
아 대괄호안에 계산된 값을 써줘야 하는군요! 덕분에 잘 적용 되었습니다. 감사합니다 진성님 :)
